tests/test_partisn.py

```diff
@@ -558,6 +560,11 @@ def write_partisn_input_with_names_dict():
 def test_write_partisn_input_with_names_dict():
     """Test full input file creation for 1D case with a names_dict provided
     """
+    try:
+        from pyne import dagmc
```

Comment:
Here they are. Given the body of these functions, I'd say that moving these blocks into the corresponding write functions would be more clear.

Reply:
Unless that doesn't give the behavior we want when running the tests?